### PR TITLE
add welcome post and first content post

### PR DESCRIPTION
**Pull request by timbroder**
This pr:

* Completes #7 by adding _Pasting an Image Into Markdown_
* Adds a welcome post
* Cleans out the default posts our theme came with

### CODE REVIEW

File: _posts/2017-09-26-welcome.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+layout: post
+title:  "Welcome to Building Kidfund!"
+date:   2017-09-26 13:00:00
+author: tim
+category: Blog
+---
+
+Welcome to [building.kid.fund](http://building.kid.fund/ "building.kid.fund")! 
+
+We'd like to use this blog to share everything we can about how we build Kidfund or talk through problems we may be encountering. Topics will cover everything from design decisions, to technical architecture and code, to process, daily hacks, and anything else we think may be helpful.
```
Comment (luisfcolon) [inline _posts/2017-09-26-welcome.md]: __Suggestion__

At Kidfund we are super busy building and coding our app. We'd love to share some of our experiences with you. So we're making a blog about it. 

Topics will cover everything from design decisions, technical architecture, code, process, daily hacks, and anything else we think may be helpful.

@mrh-is What do you think? We need something light and fun!

Comment (timbroder) [reply _posts/2017-09-26-welcome.md]: I like it, @mrh-is any thoughts before I make the change?

Comment (mrh-is) [reply _posts/2017-09-26-welcome.md]: Yeah, love the direction @luisfcolon! I changed the language slightly to be more conversational and more directly speak to what the reader gets out of this.
>At Kidfund, we're always busy designing, coding, and shipping our app. Along the way, we keep finding ideas & tips that we think could help others. So we're making a blog to share them!
>
>We'll cover everything from design decisions, technical architecture, code, process, daily hacks, and anything else we think might help you out.

Comment (luisfcolon) [reply _posts/2017-09-26-welcome.md]: @mrh-is @timbroder 

Yasssssss. Love it.

**Comment by timbroder** [comment]
👍

**Pull request merged**